config/environment-importer.php

```diff
@@ -108,6 +108,18 @@
 
     'sensitive_tables' => [],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Environment Tables
+    |--------------------------------------------------------------------------
+    |
+    | Here you can define any tables that should have their data persist when
+    | dumping the database.
+    |
+    */
+
+    'environment_tables' => [],
```

Comment:
I'm wondering if we should rename the two `*_tables` config keys, for example to `skip_tables` and `persist_tables`? Now I look at the current names they don't feel 100% clear to me, but maybe I'm being complicated. What do you think?

Reply:
Sounds good to me!